guides/doc-Managing_Hosts/topics/proc_registering-a-host.adoc

```diff
@@ -33,83 +33,97 @@ Optional: If the *Registration* feature is not enabled on your {SmartProxy}, ent
-. From the *Operating System* list, select the operating system of hosts that you want to register.
-. Optional: From the *{SmartProxy}* list, select the {SmartProxy} to register hosts through. You must select the internal {SmartProxy} if you do not want to use an external {SmartProxy}.
-. Optional: *Insecure* - This makes the first call insecure, however, during this first call, hosts download the CA file from {Project}. Hosts will use this CA file to connect to {Project} with all future calls making them secure.
+Fields that inherit value from *Host group*: Operating system, Activation Key(s) and Lifecycle environment.
+. Optional: From the *{SmartProxy}* list, select the {SmartProxy} to register hosts through.
+ifdef::satellite[]
+You must select the internal {SmartProxy} if you do not want to use an external {SmartProxy}.
+endif::[]
+. Optional: From the *Operating system* list, select the operating system of hosts that you want to register.
+ifdef::satellite[]
+Specifying an operating system is required when you register machines without subscription-manager, such as Debian or Ubuntu.
+endif::[]
+. Optional: Select the *Insecure* option, if you want to make the first call insecure.
+During this first call, hosts download the CA file from {Project}.
+Hosts will use this CA file to connect to {Project} with all future calls making them secure.
 +
 If an attacker, located in the network between {Project} and a host, fetches the CA file from the first insecure call, the attacker will be able to access the content of the API calls to and from the registered host and the JSON Web Tokens (JWT).
 Therefore, if you have chosen to deploy SSH keys during registration, the attacker will be able to access the host using the SSH key.
 +
-
-. From the *Setup REX* list, select whether you want to deploy {Project} SSH keys to hosts or not.
+The following is an example of the `curl` command with the `--insecure` option:
 +
-If set to `Yes`, SSH keys will be installed on the registered host. The inherited value is based on the `host_registration_remote_execution` parameter.
-It can be inherited e.g. from host group, operating system, organization. When overridden, the selected value will be stored on host parameter level.
+[options="nowrap", subs="+quotes,attributes"]
+----
+curl -sS --insecure https://{foreman-example-com}/register...
+----
++
+If you do not want to call the `curl` command with the `--insecure` option, you can manually copy and install the CA file on each host.
++
+To do this, find where {Project} stores the CA file by navigating to *Administer* > *Settings* > *Authentication* and locating the value of the *SSL CA file* setting.
++
+Copy the CA file to the `/etc/pki/ca-trust/source/anchors/` directory on hosts and enter the following commands:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+# update-ca-trust enable
+# update-ca-trust
+----
```

Comment:
Allright, I was too fast. Well, in that case make this the default and move it above. Avoid mentioning `--insecure` please, we do not want users to do insecure setups.

Reply:
@lzap In that case, why do we have it in the WebUI in the first place?

Reply:
I will do the changes you requested, I'm just saying it might be something to consider in UX.

Reply:
> why do we have the Insecure option in the WebUI in the first place?

Good question. We should users encourage to not use the `insecure`, because it is insecure (duh!).

With `--insecure` flag, they just need to run registration command and host is registered.  It is simple and easy.
Without `insecure`, users have to prior the registration connect to the hosts, deploy the certificates and run `update-ca-trust`. After that they can run the registration command and register the host.

They have to choose between being comfortable and being secure.